docs/cloudcost/usage.md

```diff
@@ -2,6 +2,60 @@
 
 ## Detect expensive API
 
-## Analyze as part of user session
+The Dashboard Screen provides a consolidated view of CPU 
+and Storage usage, along with traffic and activity. 
+Below the main dashboards, you'll find a detailed breakdown of 
+these metrics for each API. 
 
-## Analyze trace 
\ No newline at end of file
+![dashboard-view](../images/cco-dashboard-view.png)
+
+For each API call, you can see a breakdown by Direct and Indirect calls. 
+Direct calls are straightforward API requests, while Indirect calls might 
+involve multiple steps and dependencies. Additionally, you can view specific 
```

Comment:
```
while Indirect calls might involve multiple steps and dependencies
```

I don't get it. Let's say something like:
"and indirect which represent all side effects as asyncrously executed methods"